DataFormats/Detectors/ITSMFT/MFT/include/DataFormatsMFT/TrackMFT.h

```diff
@@ -70,6 +70,24 @@ class TrackMFT : public o2::track::TrackParCovFwd
   const o2::track::TrackParCovFwd& getOutParam() const { return mOutParameters; }       ///< Returns track parameters fitted outwards
   void setOutParam(const o2::track::TrackParCovFwd parcov) { mOutParameters = parcov; } ///< Set track out parameters
 
+  void setClusterSize(int l, int size)
+  {
+    if (l >= 11) {
```

Comment:
Why 11 and not 10?

Reply:
@shahor02 indeed you are right,  the index of the layer in never larger than 10. I commit this modification